docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-assist.md

```diff
@@ -12,6 +12,10 @@ menu:
 type: docs
 ---
 
+{{<tip title="v2.20 and earlier">}}
```

Comment:
The "v" in v2.20 gets capitalized, not such a big deal.. looks a bit odd. Maybe change it for "For v2.20 and earlier"